src/app/helps/create/components/RequestConfirmation/index.js

```diff
@@ -0,0 +1,251 @@
+import { RequestHeader } from "../RequestHeader";
+import styles from "../styles/global.module.css";
+import { ComponentFormState } from "..";
+import { useFormContext } from "react-hook-form";
+import { computedFieldProps } from "../../utils/computeFieldProps";
+import {
+  PlaceOutlined,
+  StarOutlined,
+  StartOutlined,
+} from "@mui/icons-material";
+export const RequestConfirmation = ({ step }) => {
+  const { getValues } = useFormContext();
+
+  const data = getValues();
+  return (
+    <Box>
+      <div>
+        <RequestHeader step={step} />
+        <Grid container columns={12}>
+          <Grid item xs={12}>
+            <div>
+              <Box
+                sx={{
+                  display: {
+                    md: "flex",
+                    xs: "block",
+                  },
+                }}
+              >
+                <div
+                  style={{
+                    width: "100%",
+                  }}
+                >
+                  <Typography
+                    variant="h6"
+                    fontWeight={400}
+                    style={{
+                      display: "flex",
+                      alignItems: "center",
+                      gap: 10,
+                    }}
+                  >
+                    <StarOutlined color="primary" />
+                    <span> عن طلب المساعدة</span>
+                  </Typography>
+                  <Grid
+                    columns={{
+                      md: 6,
+                      xs: 12,
+                    }}
+                    container
+                  >
+                    <Grid item xs={12} md={3} mt={3}>
+                      <Typography
+                        className={styles.helpInfoLabel}
+                        variant="body2"
+                      >
+                        المدينة أو الجهة
+                      </Typography>
+                      <Typography className={styles.helpInfoText}>
+                        <span>{data.city}</span>
+                        {data.longitude && (
+                          <a
+                            href={`https://www.google.com/maps/search/?api=1&query=${data.latitude},${data.longitude}`}
+                            target="_blank"
+                          >
+                            <Button
+                              variant="outlined"
+                              style={{
+                                fontSize: 13,
+                                display: "flex",
+                                alignItems: "center",
+                                marginRight: 7,
+                                fontWeight: "bold",
+                                color: "#0E6146",
+                                marginTop: 10,
+                                gap: 5,
+                              }}
+                            >
+                              <PlaceOutlined />
+                              <span>افتح خريطة الموقع</span>
```

Comment:
we will need this btn inside the help not here.

Reply:
I understand. It seems there was an unintended addition on the confirmation screen. Nonetheless, it's beneficial for UX, offering a full request preview and ensuring the location points correctly before the submission?

Reply:
make sense